File: scripts/talker.py
#!/usr/bin/env python
import rospy
from std_msgs.msg import Int64
from study_pkg.msg import Control

rospy.init_node('talker')
pub = rospy.Publisher('my_chat_topic', Control, queue_size=10)
rate = rospy.Rate(10)

def start_talker():
    msg = Control()
    msg.steer = 40
    msg.speed = 1
    while not rospy.is_shutdown():
        msg.steer += 1
        rospy.loginfo(str(msg.steer) + ' ' + str(msg.speed)) # output to terminal
        pub.publish(msg)        # publish to topic
        rate.sleep()            # sleep rate^(-1) second's

try:
    start_talker()
except (rospy.ROSInterruptException, KeyboardInterrupt):
    rospy.logerr('Exception catched')

File: src/talker.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>
#include <study_pkg/Control.h>

#include <sstream>

int main(int argc, char **argv)
{
  ros::init(argc, argv, "cpp_talker");

  ros::NodeHandle n;

  ros::Publisher pub = n.advertise<study_pkg::Control>("cpp_chatter", 1000);

  ros::Rate loop_rate(1);

  int count = 0;
  while ( ros::ok() )
  {
    study_pkg::Control msg;
    msg.steer = 20;
    msg.speed = 10;

    ROS_INFO("Transmited from cpp: speed = %d, steer = %d", msg.speed, msg.steer);

    pub.publish(msg);

    ros::spinOnce();

    loop_rate.sleep();
  }

  return 0;
}
